test/testdata/lsp/code_actions/extract_class_func.rb

```diff
@@ -0,0 +1,36 @@
+# typed: strict
+# selective-apply-code-action: refactor.extract
+
+module Foo
+  extend T::Sig
+
+  sig {returns(String)}
+  def self.greeting
+         # ^^^^^^^^ apply-code-action: [A] Extract method to module
+    'Hello'
+  end
```

Comment:
Do you want to add a test where the indentation level is mismatched?

E.g., if you do

```ruby
module Foo
  module Bar
    module Qux
      def self.greeting; end
    end
  end
end
```

and then try to move this into a new module, it'll have super extra indentation. I think that's fine, the user can always reindent at their leisure, but I'd love a test.

Reply:
By the way how it's implemented at the moment the new module would not have an extra indentation, but I like the idea of testing single line methods and methods without sigs.

Reply:
While I was adding a test for a single line method, I actually found a bug, thanks for pointing to this!